docs/modules/converter/pages/syntax/auto-animate.adoc

```diff
@@ -0,0 +1,152 @@
+= Auto Animate
+
+This feature is a wrapper of reveal.js auto animation.
+
+Since your adoc file will be transformed into HTML slide file, 
+all settings will be mapped to `data-*` HTML attributes.
+
+For bool attributes which means true if you set it, or false if you omit it,
```

Comment:
I think it's misleading to talk about "attributes" since we don't know if we are talking about Asciidoctor attributes or reveal.js attributes.

I think we should talk about reveal.js settings and keep the term attributes when we are referring to Asciidoctor attributes.

Reply:
HTML property is used to distinguish from asciidoctor attributes.